pkg/sql/catalog/systemschema/system.go

```diff
@@ -1564,6 +1564,41 @@ var (
 var (
 	// LeaseTable is the descriptor for the leases table.
 	LeaseTable = func() SystemTable {
+		return makeSystemTable(
+			LeaseTableSchema,
+			systemTable(
+				catconstants.LeaseTableName,
+				keys.LeaseTableID,
+				[]descpb.ColumnDescriptor{
+					{Name: "descID", ID: 1, Type: types.Int},
+					{Name: "version", ID: 2, Type: types.Int},
+					{Name: "nodeID", ID: 3, Type: types.Int},
+					{Name: "sessionID", ID: 4, Type: types.Bytes},
+					{Name: "crdb_region", ID: 5, Type: types.Bytes},
+				},
+				[]descpb.ColumnFamilyDescriptor{
+					{
+						Name:        "primary",
+						ID:          0,
+						ColumnNames: []string{"descID", "version", "nodeID", "sessionID", "crdb_region"},
+						ColumnIDs:   []descpb.ColumnID{1, 2, 3, 4, 5},
+					},
+				},
+				descpb.IndexDescriptor{
+					Name:           "primary",
+					ID:             3,
+					Unique:         true,
+					KeyColumnNames: []string{"crdb_region", "descID", "version", "sessionID", "nodeID"},
+					KeyColumnDirections: []catenumpb.IndexColumn_Direction{
+						catenumpb.IndexColumn_ASC, catenumpb.IndexColumn_ASC, catenumpb.IndexColumn_ASC,
+						catenumpb.IndexColumn_ASC, catenumpb.IndexColumn_ASC,
+					},
+					KeyColumnIDs: []descpb.ColumnID{5, 1, 2, 4, 3},
+				},
+			))
+	}
+	// V23_1_LeaseTable is the descriptor for the leases table.
```

Comment:
nit: better comments for this table (currently it's the same as the lease table above).

also, what's our plan about this `V23_1_LeaseTable`? Should we add a comment like "TODO (faizan): remove this table on V24_1"?